src/components/avroToTypescript/RecordConverter.ts

```diff
@@ -8,13 +8,93 @@ import {PrimitiveConverter} from "./PrimitiveConverter";
 
 export class RecordConverter extends BaseConverter {
 
+    public convertRecordToClass( recordType: RecordType ): ExportModel {
+        const classRows: string[] = [];
+        const interfaceRows: string[] = [];
+        const importRows: string[] = [];
+        const importExportModel: ExportModel = new ExportModel();
+        const classExportModel: ExportModel = new ExportModel();
+        const interfaceExportModel: ExportModel = new ExportModel();
+        const interfacePostFix = "Interface";
+        const TAB = SpecialCharacterHelper.TAB;
+
+        importRows.push(`// tslint:disable`);
+        importRows.push(`import {BaseAvroRecord} from "../../BaseAvroRecord";`);
+
+        classRows.push(`export class ${recordType.name} extends BaseAvroRecord implements ${recordType.name}Interface {`);
+        interfaceRows.push(`export interface ${recordType.name}${interfacePostFix} {`);
+
+        classRows.push(``);
+
+        classRows.push(`${TAB}public static readonly subject: string = "${recordType.name}";`);
+        classRows.push(`${TAB}public static readonly schema: object = ${JSON.stringify(recordType, null, 4)}`);
+
+        classRows.push(``);
+
+        classRows.push(`${TAB}public static deserialize(buffer: Buffer, newSchema?: object): ${recordType.name} {`);
+        classRows.push(`${TAB}${TAB}const result = new ${recordType.name}();`);
+        classRows.push(`${TAB}${TAB}const rawResult = this.internalDeserialize(buffer, newSchema);`);
+        classRows.push(`${TAB}${TAB}result.loadValuesFromType(rawResult);`);
+        classRows.push(``);
+        classRows.push(`${TAB}${TAB}return result;`);
+        classRows.push(`${TAB}}`);
+
+        classRows.push(``);
+
+        recordType.fields.forEach((field: Field) => {
+            const fieldType = `${this.getField(field)}`;
+            const interfaceRow = `${SpecialCharacterHelper.TAB}${fieldType};`;
```

Comment:
Just "TAB" would be better here to be consistent with the rest of the usages in this file